src/content/docs/en/recipes/build-custom-img-component.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: Build a custom image component 
+description: Learn how to build a custom image component that supports media queries using the getImage function
+i18nReady: true
+type: recipe
+---
+
+Astro provides a couple of built in components that you can use to display and optimize your images.  The `<Picture>` component allows you to display responsive images and work with different formats and sizes. The `<Image>` componet will optimze your images and allow you to pass in different formats and quality properties.
+
+But there will be times where you will need to have more customizable options for your images that the `<Picture>` and `<Image>` components currently do not support. 
+
+In this recipe, you will learn how use the `getImage()` function to create your own custom image component that supports the use of media queries/art direction.  The purpose of the `getImage()` function is to create images for use outside of direct embedding in HTML.
+
+## Recipe
+
+1. Create a new Astro component and import the `getImage()` function
+
+    ```astro title="src/components/MyCustomImageComponent.astro"
+    ---
+     import { getImage } from "astro:assets";
+    ---
+
+    ```
+
+2. Destructure the `props` from `Astro.props`. You can also use TypeScript to type the props. 
+
+   ```astro title="src/components/MyCustomImageComponent.astro"
+    ---
+    import type { ImageMetadata } from "astro";
+    import { getImage } from "astro:assets";
+
+    interface Props {
+        mobileImgUrl: string | ImageMetadata;
+        desktopImgUrl: string | ImageMetadata;
+        alt: string;
+    }
+
+    const { mobileImgUrl, desktopImgUrl, alt } = Astro.props;
+    ---
+
+    ```
+
+3. Call the `getImage()` function and pass in the properties you want to use into the options object 
+
+    ```astro title="src/components/MyCustomImageComponent.astro"
+    ---
+    import type { ImageMetadata } from "astro";
+    import { getImage } from "astro:assets";
+
+    interface Props {
+        mobileImgUrl: string | ImageMetadata;
+        desktopImgUrl: string | ImageMetadata;
+        alt: string;
+    }
+
+    const { mobileImgUrl, desktopImgUrl, alt } = Astro.props;
+
+    const mobileImg = await getImage({
+        src: mobileImgUrl,
+        format: "webp",
+        width: 200,
+        height: 200,
+    });
+
+    const desktopImg = await getImage({
+        src: desktopImgUrl,
+        format: "webp",
+        width: 800,
+        height: 200,
+    });
+    ---
+
+    ```
+
+4. Add the `source` and `img` elements inside the `picture` element and then set your desired media queries. 
```

Comment:
```suggestion
4. Create a `<picture>` element that generates `srcset` with your different images based on your desired media queries. 
```
If the picture element is a new thing, I always like instructions that say, K, you gotta make a picture element with.... I know these are tiny word changes, but when people read "THE" thing, it can stress them out if they don't know exactly what that is. ("THE picture element? Wait, WHAT picture element? I don't know what that is?) Sometimes even just choosing to write "a" instead of "the" feels less like, "I'm supposed to know this exact thing, and I don't."